x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -72,7 +82,14 @@ export const HomePage: React.FC<HomePageProps> = ({ children }) => {
     <WrappedByAutoSizer data-test-subj="wrapped-by-auto-sizer" ref={measureRef}>
       <HeaderGlobal />
 
-      <Main data-test-subj="pageContainer">
+      <Main
+        data-test-subj="pageContainer"
+        $globalFullScreen={globalFullScreen}
+        top={getPageContainerTop({
```

Comment:
please `useMemo` for the value

Reply:
per our out-of-band discussion, i'll assign this to a non-memoized variable for readability